assets/src/edit-story/app/snackbar/snackbar.js

```diff
@@ -0,0 +1,133 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const Container = styled.div`
+  display: flex;
+`;
+
+const Main = styled.div`
+  background-color: ${({ theme }) => theme.colors.bg.v0};
+  border-radius: 4px;
+  color: ${({ theme }) => theme.colors.fg.v1};
+  max-width: 456px;
+  display: flex;
+  flex-grow: 1;
+  justify-content: center;
+  align-items: baseline;
+  padding: 0;
+  margin-bottom: ${({ position }) =>
+    position.indexOf('bottom') === 0 ? '0.5em' : 0};
+  margin-top: ${({ position }) =>
+    position.indexOf('top') === 0 ? '0.5em' : 0};
+  font-size: ${({ theme }) => theme.fonts.body2.size};
+  line-height: ${({ theme }) => theme.fonts.body2.lineHeight};
+  flex-direction: row;
+`;
+
+const Content = styled.div`
+  padding: 16px;
+  flex-grow: 1;
+`;
+
+const BoldSpan = styled.span`
+  font-weight: bold;
+`;
+
+const List = styled.ul`
+  list-style: disc;
+  margin-left: 20px;
+  font-weight: bold;
+  margin-bottom: 0;
+`;
+
+const ListItem = styled.li``;
+
+const ActionButton = styled.button`
+  background: transparent;
+  display: flex;
+  justify-content: center;
+  font-size: ${({ theme }) => theme.fonts.body2.size};
+  line-height: ${({ theme }) => theme.fonts.body2.lineHeight};
+  color: ${({ theme }) => theme.colors.action};
+  outline: none;
+  border: 0;
+  cursor: pointer;
+  padding: 16px;
+`;
+
+function Snackbar({ notification, position }) {
+  const notificationMessage = () => {
+    if (notification.type === 'error') {
+      if (notification.data === 'ValidError') {
+        return (
+          <Content>
+            {'Please choose only '}
+            <BoldSpan>{'jpg, jpeg, png, svg, gif'}</BoldSpan>
+            {' or '}
+            <BoldSpan>{'webp'}</BoldSpan>
+            {' to upload'}
+          </Content>
+        );
+      } else if (notification.multiple) {
+        return (
+          <Content>
+            {'Please choose only '}
+            <BoldSpan>{'jpg, jpeg, png, svg, gif'}</BoldSpan>
+            {' or '}
+            <BoldSpan>{'webp'}</BoldSpan>
+            {' to upload. The following file failed to uploaded:'}
+            <List>
+              {notification.data.map((item) => (
+                <ListItem key={item}>{item}</ListItem>
+              ))}
+            </List>
+          </Content>
+        );
+      }
+      return <Content>{notification.message}</Content>;
+    }
+    return <Content>{notification.message}</Content>;
+  };
+  return (
+    <Container positon={position}>
+      <Main position={position}>
+        {notificationMessage()}
+        {notification.retryAction && (
+          <ActionButton onClick={notification.retryAction}>
+            {'RETRY'}
```

Comment:
```suggestion
            {__('Retry', 'web-stories')}
```

If it needs to be capitalized, we should do so via CSS.

Reply:
Fixed